charts/datasaur/templates/start/elasticsearch-eck.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.elasticsearch.enabled }}
+apiVersion: elasticsearch.k8s.elastic.co/v1
+kind: Elasticsearch
+metadata:
+  name: {{ .Release.Name }}-elasticsearch
+  annotations:
+    "helm.sh/hook": post-install,post-upgrade
+spec:
+  version: 8.10.4
+  http:
+    tls:
+      selfSignedCertificate:
+        disabled: true
+  nodeSets:
+  - name: master-nodes
+    count: 3
```

Comment:
is the minimum node for ES running well 3?

Reply:
yes, karna master nya harus ganjil dan lebih besar dari 1, jadi ya minimal 3 @tonoman3g